src/Object/Post.php

```diff
@@ -412,7 +417,7 @@ public function getTemplateData(array $conv_responses, $thread_level = 1)
 			'sparkle'         => $sparkle,
 			'title'           => $title_e,
 			'localtime'       => DateTimeFormat::local($item['created'], 'r'),
-			'ago'             => $item['app'] ? L10n::t('%s from %s', Temporal::getRelativeDate($item['created']), $item['app']) : Temporal::getRelativeDate($item['created']),
+			'ago'             => $item['app'] ? L10n::t('%s from %s', $ago, $item['app']) : $ago,
```

Comment:
Is it really `$item['app']` or should this be an `ago`?

Reply:
It is really "app". It's for "three minutes ago by Twidere"

Reply:
The display of the app information is theme dependent I think :-D in vier only the time is shown, in frio also the app.